Cargo.toml

```diff
@@ -26,3 +26,7 @@ cursor-icon = { version = "1.0.0", default-features = false, optional = true }
 log = { version = "0.4.17", optional = true }
 memchr = { version = "2.7.4", default-features = false }
 serde = { version = "1.0.160", features = ["derive"], optional = true }
+
+[package.metadata.docs.rs]
+features = ["ansi"]
```

Comment:
Ansi isn't the only feature, so I'm not sure it's correct to provide only ansi here?

Reply:
Do you mean that we should add `serde` ?
https://github.com/alacritty/vte/pull/137#issuecomment-2768132829
> I see only serde but it doesn't seem relevant to me.

Reply:
I'm not sure how these features are used, but if they're just listed on docs.rs, then all features should be included.

Reply:
> I'm not sure how these features are used, but if they're just listed on docs.rs, then all features should be included.

https://github.com/alacritty/vte/pull/137/commits/cba10b84882939079ebde2f18509afeeecabb4ba